bin/setup

```diff
@@ -5,6 +5,9 @@ set -e
 gem install bundler --conservative
 bundle check || bundle install
 
+# Add binstubs to PATH via export PATH=".git/safe/../../bin:$PATH" in ~/.zshenv
+mkdir -p .git/safe
```

Comment:
It doesn't make much sense to use this approach (trusting a repository) if you do it automatically. This should be a deliberate choice by each person to trust the repo after examining the code and bins.